File: Java/1008.java
import java.util.Scanner;

public class 1008 {
	public static void main(String[] args) {
	    Scanner read = new Scanner(System.in);
	    int num, time;
	    double byTime;
	    num = read.nextInt();
	    time = read.nextInt();
	    byTime = read.nextDouble();
	    System.out.printf("NUMBER = %d\n", num);
	    System.out.printf("SALARY = U$ %.2f\n", (double)time * byTime);
	}
}

File: Go/1008.go
package main

import (
    "fmt"
)

func main () {
    var num, time int
    var byTime float64
    fmt.Scan(&num, &time, &byTime)
    fmt.Printf("NUMBER = %d\n", num)
    fmt.Printf("SALARY = U$ %.2f\n", float64(time) * byTime)
}
